src/test/core.ts

```diff
@@ -145,6 +147,7 @@ describe('snabbdom', function () {
       assert(!elm.classList.contains('not'));
     });
     it('receives classes in selector when namespaced', function () {
+      if (!hasSvgClassList) return
```

Comment:
Why `return` not `skip`?

Reply:
What API includes, `skip`, please? Is that a mocha thing?

Reply:
https://mochajs.org/#inclusive-tests, `this.skip(...)`

Reply:
Good tip. Done.